debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/mysql/generated/MySqlParser.g4

```diff
@@ -2045,10 +2045,10 @@ constant
 //    Data Types
 
 dataType
-    : typeName=(
+    : (typeName=(
       CHAR | CHARACTER | VARCHAR | TINYTEXT | TEXT | MEDIUMTEXT | LONGTEXT
-       | NCHAR | NVARCHAR
-      )
+       | NCHAR | NVARCHAR | LONG
+      ) | typeName=LONG VARCHAR)                                    // LONG VARCHAR is the same as LONG
```

Comment:
LONG VARCHAR equals MEDIUMTEXT

Reply:
See https://dev.mysql.com/doc/refman/8.0/en/other-vendor-data-types.html
`LONG` == `LONG VARCHAR` and this maps to `MEDIUMTEXT` - so I opted for easier grammar as as soon as you'll get `LONG` it does not matter if there is `VARCHAR` or not